README.md

```diff
@@ -33,15 +33,14 @@ For help using Keplr Wallet, Visit our [User Support Site](https://help.keplr.ap
   ```
   
 - [Node.js v18+](https://nodejs.org/)
-- [Yarn v3+](https://yarnpkg.com/getting-started/install)
   
 Clone this repo and run:
 
 ```sh
 yarn && yarn build
 ```
 
-Browser extension's build output is placed in `packages/extension/build/manifest-v3`, and you can check out [this page](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked) for installing the developing version.
+Browser extension's build output is placed in `packages/extension/build/manifest-v3`. this output is only worked on Chrome so we recommend other build output(in `packages/extension/build/manifest-v2`, `packages/extension/build/firefox` for Firefox) for other browser. you can check out [this page](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked) for installing the developing version.
```

Comment:
You can find the build output of Keplr Extension in `packages/extension/build/manifest-v3`. This output only works on Chrome, so we recommend using other build outputs (in `packages/extension/build/manifest-v2` or `packages/extension/build/firefox`) for other browsers. You can visit [this page](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked) for instructions on installing the development version.